Sources/Logging/Logging.swift

```diff
@@ -38,21 +38,75 @@ import WASILibc
 /// ```swift
 /// logger.info("Hello World!")
 /// ```
+
 public struct Logger {
+    #if compiler(>=5.7)
+        /// Storage class to hold the label and log handler/
+        @usableFromInline
+        internal final class Storage: @unchecked /* and not actually */ Sendable /* but safe if only used with CoW */ {
```

Comment:
I think we can make this easier by just moving the conformance to the bottom and compiler guard it like this
```
#if compiler(>=5.7)
extension Logger.Storage: @unchecked Sendable {}
#endif
```